lib/public/components/Filters/common/filtersPanelPopover.js

```diff
@@ -32,20 +32,28 @@ const filtersToggleTrigger = () => h('button#openFilterToggle.btn.btn.btn-primar
  *
  * @return {Component} the filters panel
  */
-const filtersToggleContent = (filteringModel, columns) => h('.w-l.scroll-y', [
-    h('.f4.ph3.pv2', 'Filters'),
-    Object.entries(columns)
-        .filter(([_, column]) => column.filter)
-        .map(([columnKey, column]) => [
-            h(`.flex-row.items-baseline.ph3.pv1.${columnKey}-filter`, [
-                h('.w-30.f5', column.name),
-                h('.w-70', typeof column.filter === 'function' ? column.filter(filteringModel) : column.filter),
+const filtersToggleContent = (filteringModel, columns) => h('.w-l.scroll-y.flex-column.p3.g3', [
+    h('.flex-row.items-center.g3.justify-between', [
```

Comment:
```suggestion
    h('.flex-row.justify-between', [
```
As you are using `justify-between`, there is no point in also putting gap between elements and placing them on center